apps/expo/src/screens/create-question/index.tsx

```diff
@@ -94,6 +106,50 @@ export const CreateQuestionScreen: FC = () => {
         })),
   );
 
+  const bottomSheetRef = useRef<BottomSheet>(null);
+
+  const snapPoints = useMemo(() => ["5%", "25%", "60%"], []);
+
+  const openBottomSheet = () => {
+    bottomSheetRef.current?.expand();
+  };
+
+  const goToCreateQuestion = () => {
+    if (isLastQuestionInEdit()) {
+      setLastIndex();
+      setTimeLimitOptions(TIME_LIMIT_OPTIONS);
+      setPointOptions(POINT_OPTIONS);
+      setQuestionTitle("");
+      setChoices(
+        question!.type === "multiple_choice"
+          ? question!.choices.map((_, idx) => ({
```

Comment:
Returns an error:

"Cannot read property of 'type' of undefined"

Please fix first